nni/tools/jupyter_extension/management.py

```diff
@@ -23,8 +34,21 @@ def install():
     _backend_config_file.write_text(json.dumps(_backend_config_content))
 
     _frontend_dst.parent.mkdir(parents=True, exist_ok=True)
-    shutil.copytree(_frontend_src, _frontend_dst)
+
+    if jupyter_lab_version == '2':
+        _v2_backend_config_file.parent.mkdir(parents=True, exist_ok=True)
+        _v2_backend_config_file.write_text(json.dumps(_v2_backend_config_content))
+
+        if (os.path.islink(_frontend_src)):
+            linkto = os.path.realpath(_frontend_src)
```

Comment:
For consistency of code style, please use `_frontend_src.is_symlink()`.

Reply:
changed